include/rocksdb/sst_file_reader.h

```diff
@@ -5,22 +5,60 @@
 #pragma once
 #ifndef ROCKSDB_LITE
 
-#include "rocksdb/sst_dump_tool.h"
-
+#include <inttypes.h>
 #include <memory>
 #include <string>
-#include "db/dbformat.h"
-#include "options/cf_options.h"
-#include "util/file_reader_writer.h"
+#include <vector>
+
+#include "rocksdb/env.h"
+#include "rocksdb/options.h"
+#include "rocksdb/table_properties.h"
+#include "rocksdb/types.h"
 
 namespace rocksdb {
 
+struct ImmutableCFOptions;
+class RandomAccessFileReader;
+class InternalKeyComparator;
+class TableReader;
+struct TableBuilderOptions;
+
+struct DefaultKvHandler {
```

Comment:
I think these `DefaultXXHandler` should be moved out of this file.

Reply:
Move it to another file?